Makefile.am

```diff
@@ -264,6 +264,9 @@ dummy_man_pages = \
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libfabric.pc
 
+libfabric.map: libfabric.map.cpp
+	@$(CPP) $(CPPFLAGS)  -P -C $< -o $@
```

Comment:
Yes it can. We do so in ompi. Look in opal/mca/hwloc/hwloc.h, for example. It might need to be the *entire*file name, tho - not sure if you can use ## operator to care are strings into a file name that #include can use.